### DISCUSSION TITLE
Notice2 Preview. Submitted -> Publishing -> Published

### DISCUSSION THREAD

**Discussion by elerik** [OP-TED/eForms-SDK#458] [Q&A] [answered]
Hi devs, 

just small question. 5.5. i submitted notice. And it still has Submitted status. I just want to know if preview will be processed like live environment  and if those notices will be updated or will they stay as "submitted" forever ?

**Comment by YvesJo** [comment, marked as answer]
Hi,
Regarding your question, I invite you to contact the TED Helpdesk: ted@publications.europa.eu , they will be able to provide you details about the workflow and status of notices. 
KR
